boards/PDM/Inc/stm32f3xx_hal_conf.h

```diff
@@ -71,7 +71,7 @@
 /*#define HAL_LPTIM_MODULE_ENABLED   */
 /*#define HAL_RNG_MODULE_ENABLED   */
 /*#define HAL_RTC_MODULE_ENABLED   */
-#define HAL_SPI_MODULE_ENABLED
+/*#define HAL_SPI_MODULE_ENABLED   */
```

Comment:
if we remove this from cube and delete the efuse code with shared SPI, can we keep shared spi in the shared folder?